src/main.rs

```diff
@@ -81,7 +69,7 @@ where
         + FnMut(SharkspotterMessage) -> Result<(), Error>,
 {
     let channel: (Sender<SharkspotterMessage>, Receiver<SharkspotterMessage>) =
-        crossbeam_channel::bounded(10);
+        crossbeam_channel::bounded(100);
```

Comment:
I don't have any issues with this per say, but just want to know why this got increased this by an order?

Reply:
This is currently only used in testing sharkspotter performance.  Considering we have upwards of 100 shards in a given region, I figured a deeper queue would keep the file writing thread a bit more busy and would reduce the amount of time the per-shard scanning threads are stuck blocked on the writing thread.

In the rebalancer PR I increased this queue depth / channel size as well.